File: Lua/Exercicio1.lua
print("Digite seu primeiro nomero: ")
local a = tonumber(io.read())

print("Digite seu segundo numero: ")
local n = tonumber(io.read())

print("Digite o numero que voce deseja saber o multiplo: ")
local x = tonumber(io.read())

for i = a, n, 1 do 
  if i % x == 0 then
   print(i)

  end
end

File: C++/Exercicio1.cpp
#include <iostream>
using namespace std;

int fib(int ns) 
{
    if (ns == 0) 
    {
        return 0;
    }
    if (ns == 1) 
    {
        return 1;
    }
        return fib(ns - 1) + fib(ns - 2);
}

int main()
{
      int a;
  
      cout << "Digite um numero: ";
  
      cin >> a;
  
      cout << fib(a) << endl;
  
  return 0;
}
